cpp/include/DataStorm/DataStorm.h

```diff
@@ -118,36 +118,6 @@ namespace DataStorm
         std::shared_ptr<DataStormI::SampleT<Key, Value, UpdateTag>> _impl;
     };
 
-    /**
```

Comment:
No longer needed, `slice2cpp` will generate this (and thankfully the implementation is the same).